tuxemon/states/control/__init__.py

```diff
@@ -0,0 +1,129 @@
+#
+# Tuxemon
+# Copyright (C) 2014, William Edwards <shadowapex@gmail.com>,
+#                     Benjamin Bean <superman2k5@gmail.com>
+#
+# This file is part of Tuxemon.
+#
+# Tuxemon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Tuxemon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Tuxemon.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Contributor(s):
+#
+# Andrew Hong <novialriptide@gmail.com>
+#
+#
+# states.start Handles the start screen which loads and creates new games
+#
+"""This module contains the Options state
+"""
+from __future__ import annotations
+\
+from functools import partial
+from typing import Any, Callable, Union, Optional
+
+import pygame
+
+from tuxemon import prepare, config
+from tuxemon.constants import paths
+from tuxemon.locale import T
+from tuxemon.menu.interface import MenuItem
+from tuxemon.menu.menu import PopUpMenu
+from tuxemon.platform.events import PlayerInput
+from tuxemon.state import State
+
+ControlStateObj = Callable[[], object]
+cfg_file = config.TuxemonConfig(paths.USER_CONFIG_PATH).cfg
+config = prepare.CONFIG
+
+def reload_control_config():
```

Comment:
keyboard control map loading is already here
https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/config.py#L131